src/main/java/org/gridsuite/loadflow/server/LoadFlowService.java

```diff
@@ -1,174 +0,0 @@
-import org.springframework.http.*;
-import org.springframework.stereotype.Service;
-import org.springframework.web.client.RestTemplate;
-import org.springframework.web.server.ResponseStatusException;
-import org.springframework.web.util.UriComponentsBuilder;
-
-import java.util.*;
-import java.util.stream.Collectors;
-
-import static org.gridsuite.loadflow.server.LoadFlowConstants.DELIMITER;
-import static org.gridsuite.loadflow.server.LoadFlowConstants.REPORT_API_VERSION;
-
-/**
- * @author Franck Lecuyer <franck.lecuyer at rte-france.com>
- */
-@ComponentScan(basePackageClasses = {NetworkStoreService.class})
-@Service
-class LoadFlowService {
-
-    @Value("${gridsuite.services.report-server.base-uri:http://report-server}/")
-    String reportServerURI;
-
-    @Value("${loadflow.default-provider}")
-    private String defaultProvider;
-
-    @Autowired
-    private NetworkStoreService networkStoreService;
-
-    @Autowired
-    private ObjectMapper objectMapper;
-
-    private static final String LOAD_FLOW_TYPE_REPORT = "LoadFlow";
-
-    private Network getNetwork(UUID networkUuid) {
-        try {
-            return networkStoreService.getNetwork(networkUuid, PreloadingStrategy.ALL_COLLECTIONS_NEEDED_FOR_BUS_VIEW);
-        } catch (PowsyblException e) {
-            throw new ResponseStatusException(HttpStatus.NOT_FOUND, e.getMessage());
-        }
-    }
-
-    private LoadFlowParameters buildLoadFlowParameters(LoadFlowParametersInfos loadflowParams, String provider) {
-        LoadFlowParameters params = loadflowParams == null || loadflowParams.getCommonParameters() == null ?
-             LoadFlowParameters.load() : loadflowParams.getCommonParameters();
-        if (loadflowParams == null || loadflowParams.getSpecificParameters() == null || loadflowParams.getSpecificParameters().isEmpty()) {
-            return params; // no specific params
-        }
-        LoadFlowProvider lfProvider = LoadFlowProvider.findAll().stream()
-                .filter(p -> p.getName().equals(provider))
-                .findFirst().orElseThrow(() -> new PowsyblException("Loadflow provider not found " + provider));
-        Extension<LoadFlowParameters> extension = lfProvider.loadSpecificParameters(loadflowParams.getSpecificParameters())
-                .orElseThrow(() -> new PowsyblException("Cannot add specific parameters with Loadflow provider " + provider));
-        params.addExtension((Class) extension.getClass(), extension);
-        return params;
-    }
-
-    LoadFlowResult run(UUID networkUuid, String variantId, List<UUID> otherNetworksUuid, LoadFlowParametersInfos loadflowParams,
-                       String provider, ReportContext reportContext) {
-        String providerToUse = provider != null ? provider : defaultProvider;
-        LoadFlowParameters params = buildLoadFlowParameters(loadflowParams, providerToUse);
-
-        // TODO: to be removed after loads merge fix in powsybl-dynaflow
```

Comment:
Warning: this hack will be deleted in your PR. It's only ok if it is deleted on main before your PR (it will be done very soon at powsybl upgrade)